src/components/dashboard/DashboardDetail.js

```diff
@@ -83,24 +83,24 @@ class DashboardDetail extends Component {
                     <DashboardActivity
                         loader={loader}
                         image={activityGreyImage}
-                        title=<Translate id='dashboard.activity' />
+                        title={<Translate id='dashboard.activity' />}
```

Comment:
Why do we need changes in this file? Seem unrelated to main topic of PR.

Reply:
touch this file to change postprocessSerdeStruct function

Reply:
I don't understand. Are you committing this because you needed to trigger auto-compile locally?

Reply:
I will change back. when I first search where actions display, I fount this file. it gives the error if without curly braces. so I change them.